sagemaker-debugger/tensorflow_profiling/low_batch_size.ipynb

```diff
@@ -119,9 +119,9 @@
    "metadata": {},
```

Comment:
<p>change "epochs to 3 to keep the training short for testing."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1856/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done.</p>